tests/integration/telemetry/stackdriver/common.go

```diff
@@ -275,6 +275,7 @@ func unmarshalFromTemplateFile(file string, out proto.Message, clName, trustDoma
 		"ClusterName":   clName,
 		"TrustDomain":   trustDomain,
 		"OnGCE":         metadata.OnGCE(),
+		"ProxyVersion":  os.Getenv("VERSION"),
```

Comment:
Not sure I understand this? Where is this set?

Reply:
I think this is depending on it being run from makefiles which is not always how the tests are run. Can we avoid this implicit dependency? We have examples like `env.IstioSrc`.

Even still, I am not sure we should do this. The version in the repo we are running the test from is not neccesarily the version we deployed and are testing. IMO we should make this a fuzzy match

Reply:
Yes, that was from the makefiles.
I use `env.ReadVersion()` now and try to remove extra version information after pulling metrics from SD.